src/AutomatedCar/AutomatedCar.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <Folder Include="Models\" />
```

Comment:
Ezt miért kell kivenni?
Egy másik PR-ben is láttam, de nem értem.

Reply:
Ez lehet hogy a vstudio volt. Ehez a fájlhoz nem nyúltam. Visszaírtam ezt a sort, hátha akkor nem tűnik el újra.